tests/testthat/test-target_market_share.R

```diff
@@ -49,15 +49,14 @@ test_that("with fake data outputs known value", {
     fake_scenario(),
     region_isos_stable
   )
-
-  expect_known_value(out, "ref-target_market_share", update = FALSE)
+  expect_snapshot("out")
```

Comment:
This `out` should be an object, not a character. This is what is causing the empty output in tests/testthat/_snaps/target_market_share.md

Reply:
Thanks!